innopoints/views/application.py

```diff
@@ -295,5 +295,14 @@ def leave_feedback(project_id, activity_id, application_id):
         log.exception(err)
         abort(400, {'message': 'Data integrity violated.'})
 
+    all_feedback_in = all(application.feedback is not None
+                          for activity in project.activities
+                          for application in activity.applications)
+    if all_feedback_in:
+        mods = [*project.moderators, project.creator]
```

Comment:
The creator is always included in the list of moderators, this would send two notifications to them

Reply:
maybe what confused me is the empty moderators list in the response of the project listing endpoint